client/client.go

```diff
@@ -36,6 +40,11 @@ var (
 	service = flag.String("service", "http://localhost:8080", "URL for deployment manager")
 	binary  = flag.String("binary", "../expandybird/expansion/expansion.py",
 		"Path to template expansion binary")
+
+	properties = flag.String("properties", "", "Properties to use when deploying a type")
+	repository = flag.String("repository",
```

Comment:
TODO, this should only have the trailing root in it. I can merge this later.
Also, consider renaming to type_registry? Or I can rename mine :)